content/source/docs/enterprise-beta/api/linkable-repos.html.md

```diff
@@ -1,53 +0,0 @@
----
-layout: enterprise2
-page_title: "Linkable Repos - API Docs - Terraform Enterprise Beta"
-sidebar_current: "docs-enterprise2-api-linkable-repos"
----
-
-# Linkable Repos API
```

Comment:
Hey, how do we do redirects right now? This URL should be redirected to the o-auth-tokens page.

Reply:
I think while in beta we don't need to because the API is going to change. Once in GA then I think we'll need to redirect; though we shouldn't have to.

Reply:
OK. 👍